requirements/edx/github.txt

```diff
@@ -76,8 +76,8 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/edx/edx-ora2.git@1.1.11#egg=ora2==1.1.11
--e git+https://github.com/edx/edx-submissions.git@1.1.1#egg=edx-submissions==1.1.1
+git+https://github.com/edx/edx-ora2.git@1.1.12#egg=ora2==1.1.12
+-e git+https://github.com/edx/edx-submissions.git@1.1.4#egg=edx-submissions==1.1.4
```

Comment:
Neither of these versions actually exist yet, so expect to see some test failures in the short-term